Accessibility-Basics.md

```diff
@@ -1,15 +1,16 @@
-## Accessibility basics: What is this thing you speak of?
+# Accessibility basics: What is this thing you speak of?
 
+##
```

Comment:
Please move this after a paragraph, the point it to show one paragraph and not the whole article when the article is very long.